packages/espree/lib/options.js

```diff
@@ -22,19 +26,30 @@ const SUPPORTED_VERSIONS = [
     15, // 2024
     16, // 2025
     17 // 2026
-];
+]);
+
+/**
+ * @typedef {typeof SUPPORTED_VERSIONS[number]} normalizedEcmaVersion
+ */
```

Comment:
Can we name this `NormalizedEcmaVersion` (with a capital N) to follow our naming conventions?